cypress/e2e/awx/views/jobs.cy.ts

```diff
@@ -98,7 +98,8 @@ describe('jobs', () => {
     });
   });
 
-  it('deletes a job from the jobs list toolbar', () => {
+  // this test is failing randomly with a 403. need to fix before reenabling
```

Comment:
I wonder if it is trying to delete a job that has already been deleted, or one that hasn't started?